src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -50,13 +58,49 @@ def get_stacktrace_string(exception: Mapping[Any, Any], event: GroupEvent) -> st
     return "\n".join(output)
 
 
+class FormattedSimilarIssuesEmbeddingsData(TypedDict, total=False):
```

Comment:
What's `total` for?

Reply:
total is to say that the keys are not required, but I realized this isn't needed after some changes I made. thanks!